### PR TITLE
V a2

### PR DESCRIPTION
**Pull request by Matthew3000**
Code examples update

### CODE REVIEW

File: examples/11.Encoder/11.Encoder.ino [hunk 1]
```diff
@@ -6,8 +6,6 @@ Dragster robot(MMAX_16_OHM);
 
 void setup() {
   pinMode(ENC1, INPUT);
-  robot.begin();
-  robot.drive(120, 0);
```
Comment (yurybotov) [inline examples/11.Encoder/11.Encoder.ino:10]: Why you remove robot.begin and .drive? A sample will not work.

**Review by yurybotov — CHANGES_REQUESTED** [review]
look comment in 11.Encoder

File: examples/11.Encoder/11.Encoder.ino (hunk 1 above)
Comment (Matthew3000) [reply examples/11.Encoder/11.Encoder.ino:10]: We changed the chapter itself. It's not convenient to play with the encoder and watch the changes in port monitor while the wheels are spinning. We added the directive to spin it manually.

Comment (yurybotov) [reply examples/11.Encoder/11.Encoder.ino:10]: Understand. OK.

**Review by yurybotov — APPROVED** [review]

**Pull request merged**